lib/curly/presenter.rb

```diff
@@ -52,6 +52,10 @@ def initialize(context, options = {})
           end
         end
 
+        if value.respond_to?(:call)
+          value = value.call
```

Comment:
The problem is, there would be no guarentee that all of the presented values would have been initialized, so the instance variables may not be set.

There is another problem - if the value _(note: not the default value)_ passed to the presenter responds to `#call`, it will call it regardless.